src/gui/nav/ViewSlider.ts

```diff
@@ -363,6 +365,9 @@ export class ViewSlider implements Component<ViewSliderAttrs> {
 			})
 			.finally(() => {
 				foregroundColumn.isInForeground = toForeground
+				if (foregroundColumn.domColumn && !toForeground) {
```

Comment:
the previous attempt by @jalhubb was similar and led to some issues, I think would be good if he could check it

Reply:
oh sorry I meant @rezbyte